cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -1037,6 +1039,9 @@ impl<I: VCodeInst> VCode<I> {
                 }
             }
 
+            // emission state is not needed anymore, move control plane back out
+            std::mem::swap(ctrl_plane, state.get_ctrl_plane());
```

Comment:
If this is forgotten, the control plane would just silently be the default one for the rest of the compilation. I guess it should be fine, (for now) it seems like this is the only place where one has to remember to move the control plane back out of the emit state.

Reply:
Yeah, we may be able to find a better way here eventually, but I think this strikes a good balance for now.